spring-cloud-dataflow-classic-docs/src/test/java/org/springframework/cloud/dataflow/server/rest/documentation/AboutDocumentation.java

```diff
@@ -35,17 +35,119 @@
 	public void getMetaInformation() throws Exception {
 		this.mockMvc.perform(get("/about").accept(MediaType.APPLICATION_JSON)).andExpect(status().isOk())
 				.andDo(this.documentationHandler.document(responseFields(
-						fieldWithPath("_links.self").description("Link to the runtime environment resource"),
+						fieldWithPath("_links.self.href").description("Link to the runtime environment resource"),
+
 						fieldWithPath("featureInfo").type(JsonFieldType.OBJECT)
 								.description("Details which features are enabled."),
+						fieldWithPath("featureInfo.analyticsEnabled").type(JsonFieldType.BOOLEAN)
+								.description("Describes if analytics feature is enabled."),
+						fieldWithPath("featureInfo.schedulerEnabled").type(JsonFieldType.BOOLEAN)
+								.description("Describes if scheduler feature is enabled."),
+						fieldWithPath("featureInfo.skipperEnabled").type(JsonFieldType.BOOLEAN)
+								.description("Describes if skipper feature is enabled."),
+						fieldWithPath("featureInfo.streamsEnabled").type(JsonFieldType.BOOLEAN)
+								.description("Describes if streams feature is enabled."),
+						fieldWithPath("featureInfo.tasksEnabled").type(JsonFieldType.BOOLEAN)
+								.description("Describes if tasks feature is enabled."),
 
 						fieldWithPath("versionInfo").type(JsonFieldType.OBJECT).description(
 								"Provides details of the Spring Cloud Data Flow Server " + "dependencies."),
+						fieldWithPath("versionInfo.implementation").type(JsonFieldType.OBJECT).description(
+								"Provides details of the implementation dependency."),
```

Comment:
We could add what implementation this is about. Same is the case for the other versionInfo attributes below as well.